package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
@sprucely how did you get this file, there isn't even a package.json in this directory?  What is it supposed to do for us?

Reply:
Wait, what? I thought I avoided staging any of the automatic package changes...

Reply:
Oh, I reverted and force pushed in order to make the message more consistent with other commits, and those changes snuck back in.

Reply:
Should I amend or commit another change to remove those?

Reply:
Just remove in another commit